src/researchhub_case/constants/case_constants.py

```diff
@@ -28,9 +29,10 @@
 RH_CASE_TYPES = [
     (AUTHOR_CLAIM, AUTHOR_CLAIM),
     (PAPER_CLAIM, PAPER_CLAIM),
+    (EXTERNAL_AUTHOR_CLAIM, EXTERNAL_AUTHOR_CLAIM),
 ]
 
 CLAIM_CONTEXT_ENTITY_TYPES = [
-    ('PAPER', 'PAPER'),
-    ('AUTHOR', 'AUTHOR'),
+    ("PAPER", "PAPER"),
```

Comment:
@le-sun will claims submitted through the paper page be considered "Paper" claims

Reply:
It will be `EXTERNAL_AUTHOR_CLAIM`. I'll update the list.